source/localizable/applications/initializers.md

```diff
@@ -34,8 +34,7 @@ export function initialize(application) {
 };
 
 export default {
-  name: 'shopping-cart',
-  initialize: initialize
+  initialize
```

Comment:
I added it like this because it's fairly common to add `before` and `after` and I wanted to maintain symmetry. I can change it though.